doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -153,8 +156,10 @@ Installing nRF Util and its commands
 
 Using the nRF54H20 DK with the |NCS| version |release| requires the following:
 
-* nRF Util version 7.11.1 or higher
-* nRF Util ``device`` version 2.7.2
+* nRF Util version 7.13.0 or higher
+* nRF Util ``device`` version 2.7.8
```

Comment:
There was bug in this version, iw ill confirm the version tomorrow.

Reply:
This had an issue with using an incorrect argument for erasing in its generated batch scripts.

``` 
Error: One or more batch tasks failed:
- [Probe] Invalid JSON: missing field `ext_mem_erase_mode`, code: Generic
```

This is fixed in `nrfutil device 2.7.10`

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/19644